build/helper.py

```
@@ -240,3 +241,119 @@ def get_rst_header_snippet(t, header_level='='):
+    end_of_line = end_of_line.translate(to_separator)
+    separator = '{0}{1}{2}'.format(start_of_line,
+                                   vertical_separator.join([x*line_marker for x in sizes]),
+                                   end_of_line)
+    # determine header separator
+    th_separator_tr = {ord('-'): '='}
+    if sys.version_info.major < 3:
+        th_separator_tr = string.maketrans('-', '=')
+    start_of_line = start_of_line.translate(th_separator_tr)
+    line_marker = line_marker.translate(th_separator_tr)
+    vertical_separator = vertical_separator.translate(th_separator_tr)
+    end_of_line = end_of_line.translate(th_separator_tr)
+    th_separator = '{0}{1}{2}'.format(start_of_line,
+                                      vertical_separator.join([x*line_marker for x in sizes]),
+                                      end_of_line)
+    # prepare result
+    table.append(separator)
+    # set table header
+    titles = data[0]
+    table.append(template.format(*titles))
+    table.append(th_separator)
+
+    for d in data[1:-1]:
+        table.append(template.format(*d))
+        if full:
+            table.append(separator)
+    table.append(template.format(*data[-1]))
+    table.append(separator)
+    return '\n'.join(table)
+
+def get_python_type_from_visa_type(visa_type):
+    if visa_type == 'ViInt32':
+        return 'integer'
+    elif visa_type == 'ViInt64':
+        return 'integer'
+    elif visa_type == 'ViUInt32':
+        return 'integer'
+    elif visa_type == 'ViUInt64':
+        return 'integer'
+    elif visa_type == 'ViReal32':
+        return 'floating point'
+    elif visa_type == 'ViReal64':
+        return 'floating point'
+    elif visa_type == 'ViChar':
+        return 'string'
+    elif visa_type == 'ViString':
+        return 'string'
+    elif visa_type == 'ViConstString':
+        return 'const string'
+    elif visa_type == 'ViBoolean':
+        return 'bool'
+    else:
+        return 'Unknown'
```

I bet you want to raise an Exception here rather than returning 'Unknown'